src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/189150887-57bc50ea-f901-4c28-846f-8df79fae8c36.png)
```suggestion
        for (int i = 0; i < NUMBER_OF_BALLS i++) {
```